src/iologindata.cpp

```diff
@@ -687,7 +687,7 @@ bool IOLoginData::saveItems(const Player* player, const ItemBlockList& itemList,
 		}
 	}
 
-	for (size_t i = 0; i < containers.size(); i++) {
+	for (size_t i = 0, n = containers.size(); i < n; i++) {
```

Comment:
Note that this is _not_ equivalent; the size of `containers` is changed (in L700) so this change breaks the code.